src/pause/140C70.c

```diff
@@ -1,17 +1,244 @@
 #include "common.h"
 
-INCLUDE_ASM(s32, "pause/140C70", func_8024D930);
+#define pause_map_tab_places_desc_string_id 0x1D00F2
```

Comment:
This should really go in `tools/splat_ext/msg.yaml` instead and referenced as `MSG_pause_map_tab_places_desc` from `message_ids.h`.

Reply:
Actually, this is the _base_ msg ID, where the following messages are in pause_map order. It may be worth naming all the messages to that effect; i.e.
```
- [0x1D, 0x0F2, pause_map_location_desc_0]
- [0x1D, 0x0F3, pause_map_location_desc_1]
- [0x1D, 0x0F4, pause_map_location_desc_2]
...
```

Reply:
I don't think the strings are consecutive due to the code that multiplies the option by 3 and then adds the base. I think it'd be best to wait to figure out what these are before creating dummy vals for them

Reply:
I recall what they are: name, 'before' desc, 'after desc'.